operations/iuf/IUF.md

```diff
@@ -641,34 +658,42 @@ HPE provides management NCN and managed node `sat bootprep` configuration files
+
+If an error is encountered while executing `iuf run`, `iuf` will attempt to complete the current stage for the other products involved. The following are strategies to recover from failures once the underlying issue has been
+resolved.
+
+### Addressing the issue without changing products
 
-It is possible to re-execute stages of an IUF session by specifying `iuf run` with the desired stage and other `iuf` arguments. If no changes were made to the product distribution files in the media directory, `iuf run` will
-re-execute any Argo steps that failed during the previous invocation of `iuf run`. Any Argo steps that previously executed successfully will be skipped if possible. If the `-f` argument is specified, all Argo steps will be
-re-executed, regardless of whether they succeeded or failed during the previous invocation of `iuf run`.
+Multiple options are available if the administrator decides to continue the install or upgrade without changing the products being installed or upgraded:
 
-### Removing a product from an IUF session
+- [`iuf resume`](#resume) can be used to re-execute the most recent `iuf run` command and continue from where the failures were encountered.
+- [`iuf restart`](#restart) can be used to re-execute the most recent `iuf run` command from the beginning of the earliest stage specified. Only failed or previously unexecuted Argo steps will be executed unless the `-f`
+  argument is specified, which forces all Argo steps to be re-executed, regardless of whether they succeeded or failed during the previous invocation of `iuf run`.
+- [`iuf run`](#run) can be used to re-execute stages of an IUF session with new `iuf` arguments. If no changes were made to the product distribution files in the media directory, `iuf run` will re-execute any Argo steps that failed
+  during the previous invocation of `iuf run`. Any Argo steps that previously executed successfully will be skipped if possible. If the `-f` argument is specified, all Argo steps will be re-executed, regardless of whether they
+  succeeded or failed during the previous invocation of `iuf run`.
+
+### Addressing the issue by removing a product
 
 If the administrator wants to remove a product from the IUF session, they must re-execute `iuf run` for the `process-media` stage with the product distribution file and uncompressed content removed from the media directory. This
 removes references to that product from the existing IUF activity.
 
-If any previously-executed stages performed operations with that product, it may be necessary to re-execute them as well to remove artifacts or metadata related to the product, e.g. to remove CFS configuration layers and rebuild
-images without that product present.
-
-### Adding a product to an IUF session
+If any previously executed stages performed operations with that product, re-execute them to remove artifacts or metadata related to the product, e.g. to remove CFS configuration layers and rebuild images without that product
```

Comment:
```suggestion
If any previously executed stages performed operations with the removed product, re-execute them. It may be necessary to perform manual operations as well, e.g. modifying the `sat bootprep` input files used to create images in order to remove references to the product.
```

Reply:
@haasken-hpe what do you think of this wording now?

Reply:
Yeah, I think that's more accurate, @deantroe.

Reply:
Fixed in next commit.